### PR TITLE
Adds "*Missing*" as a possible physical status in medical records

### PR DESCRIPTION
**Pull request by Spud5**
Inspired by this one round where a doctor asked me to list the HOS as "MIA" on the crew manifest.

image

### CODE REVIEW

**Comment by Spud5** [comment]
!review

File: html/changelogs/Spud5-physical_status_missing.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+author: Spud2
+
+
+delete-after: True
+
+
+changes:
+  - rscadd: "Adds "*Missing*" as a possible physical status in medical records."
```
Comment (SleepyGemmy) [inline html/changelogs/Spud5-physical_status_missing.yml:9]:
```suggestion
  - rscadd: "Adds \"*Missing*\" as a possible physical status in medical records."
```
escape characters necessary.

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]

**Review by SleepyGemmy — APPROVED** [review]

File: code/modules/modular_computers/file_system/programs/generic/records.dm [hunk 2]
```diff
@@ -25,7 +25,7 @@
 	var/default_screen = "general"
 	var/record_prefix = ""
 	var/typechoices = list(
-		"physical_status" = list("Active", "*Deceased*", "*SSD*", "Physically Unfit", "Disabled"),
+		"physical_status" = list("Active", "*Deceased*", "*SSD*", "*Missing*", "Physically Unfit", "Disabled"),
```
Comment (Sindorman) [inline code/modules/modular_computers/file_system/programs/generic/records.dm:28]: Why does it have `*` around it if it doesn't make it bold? What is the point of it?

**Review by Sindorman — CHANGES_REQUESTED** [review]
It looks like you also need to update `code\modules\mob\living\carbon\human\human.dm` line 679 `setmedical` variable to contain it too.

File: code/modules/modular_computers/file_system/programs/generic/records.dm (hunk 2 above)
Comment (SleepyGemmy) [reply code/modules/modular_computers/file_system/programs/generic/records.dm:28]: distinguishment. other entries have it to.

**Review by Sindorman — APPROVED** [review]

**Review by SleepyGemmy — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**